extensions/amp-autocomplete/0.1/amp-autocomplete.js

```diff
@@ -394,6 +396,7 @@ export class AmpAutocomplete extends AMP.BaseElement {
         renderedChildren.map(child => {
           child.classList.add('i-amphtml-autocomplete-item');
           child.setAttribute('role', 'listitem');
+          child.setAttribute('dir', 'auto');
```

Comment:
@sparhami you recently focused a lot on RTL, does this make sense from your perspective?

Reply:
There are some cases that this isn't what you want. For example, if your layout looked like:

```html
<div style="display: flex">
  Some suggestion
  <button class="remove-suggestion">x</button>
</div>
```

Then the side the button is on will depend on the directionality of the text, which is incorrect. It should depend on the directionality of the page (on the right for LTR languages, on the left for LTR languages), regardless of the directionality of the suggestion.

I think for plain text suggestions, it makes sense to set the directionality, but I'm not sure for template based items that it really makes sense.

Reply:
For template based items, do you know if there is existing RTL support at the time that the template service renders a template? I would think it is more reasonable to make a directionality check in the service rather than later down the line by the time it reaches autocomplete.

Reply:
I don't think there is any need for the autocomplete to set directionality, developers can do that themselves.

I think we could suggest (via documentation) for developers using templates to do something like:

```html
<div class="container">
  <div class="suggestion" dir="auto">Some suggestion</div>
  <button class="remove-suggestion">x</button>
</div>
```

Without setting the directionality from the autocomplete, the suggestion will inherit the directionality of the page.

If autocomplete were to set a `dir` here, developers could cancel out the autocomplete's `dir` by setting the `dir` explicitly on their container, but I just don't see any value for the autocomplete to set that `dir` in the first place.

Reply:
Ah, I see. Thank you for your insight!